src/Serilog.Sinks.AzureTableStorage/LoggerConfigurationAzureTableStorageExtensions.cs

```diff
@@ -63,6 +64,7 @@ public static class LoggerConfigurationAzureTableStorageExtensions
             CloudStorageAccount storageAccount,
             LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
             IFormatProvider formatProvider = null,
+            ITextFormatter textFormatter = null,
```

Comment:
To avoid binary breaking changes, the approach taken by other sinks is to create an overload of the configuration method that accepts an `ITextFormatter` as the first argument.

This also allows the `IFormatProvider` argument to be dropped from that overload, since it's not used when a text formatter is supplied explicitly.

Reply:
@nblumhardt I will create three new Extension methods that have ITextFormatter as first arg
However, IFormatProvider is still needed due to `log.RenderMessage(formatProvider)` in `LogEventEntity`

Reply:
Also when using `JsonFormatter` as the default ITextFormatter, we have the IFormatProvider around it makes sense to pass it into JsonFormatter